File: backend/src/main/java/space/davids_digital/vk_gpt_bot/integration/vk/dto/VkApiResponseDto.java
package space.davids_digital.vk_gpt_bot.integration.vk.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

public class VkApiResponseDto {
    @JsonProperty("error")
    public Error error;

    public static class Error {
        @JsonProperty("error_code")
        public int errorCode;

        @JsonProperty("error_msg")
        public String errorMsg;
    }
}

File: backend/src/main/java/space/davids_digital/sweetie/integration/vk/dto/VkApiResponseDto.kt
package space.davids_digital.sweetie.integration.vk.dto

import com.fasterxml.jackson.annotation.JsonProperty

@Deprecated("Use new API")
open class VkApiResponseDto {
    @JsonProperty("error")
    var error: Error? = null

    open class Error {
        @JsonProperty("error_code")
        var errorCode = 0

        @JsonProperty("error_msg")
        var errorMsg: String? = null
    }
}
